tests/app/test_jinja_filters.py

```diff
@@ -1006,5 +1006,124 @@ def test_summary_item_config_with_list_collector():
+                    "valueList": [{"text": "123"}],
+                },
+            ]
+        }
+    ]
+
+    result = map_summary_item_config(
+        group={
+            "blocks": [
+                {
+                    "title": "Companies or UK branches",
+                    "type": "List",
+                    "add_link": "/questionnaire/companies/add-company/?return_to=section-summary",
+                    "add_link_text": "Add another UK company or branch",
+                    "empty_list_text": "No UK company or branch added",
+                    "list_name": "companies",
+                    "related_answers": {
+                        "vmmPmD": [
+                            {
+                                "id": "edit-company",
+                                "title": None,
+                                "number": None,
+                                "question": {
+                                    "id": "add-question",
+                                    "type": "General",
+                                    "title": " ",
+                                    "number": None,
+                                    "answers": [
+                                        {
+                                            "id": "registration-number",
+                                            "label": "Registration number",
+                                            "value": 123,
+                                            "type": "number",
+                                            "unit": None,
+                                            "unit_length": None,
+                                            "currency": None,
+                                            "link": "edit_link_url",
+                                        }
+                                    ],
+                                },
+                            }
+                        ]
+                    },
+                    "answer_title": "Name of UK company or branch",
+                    "list": {
+                        "list_items": [
+                            {
+                                "item_title": "Company A",
+                                "primary_person": False,
+                                "list_item_id": "vmmPmD",
+                                "edit_link": "change_link_url",
+                                "remove_link": "remove_link_url",
+                            }
+                        ],
+                        "editable": True,
+                    },
+                }
+            ],
+        },
+        summary_type="SectionSummary",
+        answers_are_editable=True,
+        no_answer_provided="No answer Provided",
+        remove_link_aria_label="Remove Company A",
+        remove_link_text="Remove",
+        edit_link_text="Change",
+        edit_link_aria_label="Change your answer for:",
+        calculated_question={},
+    )
+
+    assert to_dict(expected) == to_dict(result)
```

Comment:
Yeah. I was just trying to keep it consistent with the other related tests

Reply:
Yeah, that makes sense, not necessarily suggesting a big change here but whats your thoughts on the tests in general?

Reply:
Jinja filters as a whole needs revisiting. Since the test name is a specific case instead of the whole config, it could be made specific to assert only what we care about. Non blocker from me.

Reply:
Yeah its non blocker for this so can look another time